samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -93,6 +94,7 @@ def mnist_classification(region='us-west-2',
 
     hpo = sagemaker_hpo_op(
         region=region,
+        endpoint_url=endpoint_url,
```

Comment:
For some of the common parameters which will be applied on all operators, we probably like to come up a different way to configure them since they don't need fine grain control.